examples/pom.xml

```diff
@@ -69,5 +69,42 @@
                 <module>integrations</module>
             </modules>
         </profile>
+        <profile>
+            <!--
+                Pipeline builds use this profile with the generate-sources phase to convert
```

Comment:
How about we move this text under the README.md inside the /examples directory ?

Reply:
Done.